src/test/scala/firrtlTests/DriverSpec.scala

```diff
@@ -355,7 +355,12 @@ class DriverSpec extends FreeSpec with Matchers with BackendCompilationUtilities
                                                  emitOneFilePerModule = true)
         }
 
-        firrtl.Driver.execute(manager)
+        firrtl.Driver.execute(manager) match {
+          case success: FirrtlExecutionSuccess =>
+            success.circuitState.annotations.length should be > (0)
+          case _ =>
```

Comment:
The test should fail if this reached right?

Reply:
The firrtl passes leave a number of annotations in the CircuitState, I think there 4 or 5 laying around in here.